agent/router/router.go

```diff
@@ -544,6 +603,11 @@ func (r *Router) GetDatacenterMaps() ([]structs.DatacenterMap, error) {
 
 	var maps []structs.DatacenterMap
 	for areaID, info := range r.areas {
+		// all consumers of this API expect only results for WAN areas so we filter
+		// out the LAN area here
```

Comment:
This comment would probably be more noticeable as part of the godoc string on the method

Reply:
I noticed your comment about filtering in the api/rpc after I had put it here so I am in the process of fixing it all up again there.